### ISSUE TITLE
Attachments ui/ux improvement request

### ISSUE THREAD

**Issue by ghost** [PrestaShop/PrestaShop#14135]
Hi!
UI/UX squad should consider dealing with product attachments in combination with languages.
Below my suggestion and use case - imho very clear and understandable

**Now:**
![attachment_improve_before](https://user-images.githubusercontent.com/8243273/59189557-a0b50680-8b7a-11e9-9454-1c5955d91a4d.jpg)

**After:**
![attachment_improve_after](https://user-images.githubusercontent.com/8243273/59189570-ae6a8c00-8b7a-11e9-9d1b-3db256f6f6b0.jpg)

Thanks for consideration in next roadmap

**Comment by khouloudbelguith** [comment]
Hi @PrestaShark,

Thanks for your report.
@marionf, @colinegin what do you think of his suggestion?

Thanks!

**Comment by TristanLDD** [comment]
Hi @PrestaShark, 
Thanks for your suggestion, it is very relevant especially in the case of instruction manuals. I think a dropdown checkbox list would be more appropriate to handles the case where many languages are installed. We'll try to work on this as soon as possible.

**Comment by marionf** [comment]
@TristanLDD 

What do you think about adding a dropdown with languages next to the fields ?

![capture d'écran_1893](https://user-images.githubusercontent.com/13449658/63168763-539b5800-c035-11e9-98d1-c32d3bc80c6a.png)

The thing is, we will also have to do it on add / edit file page

![capture d'écran_1894](https://user-images.githubusercontent.com/13449658/63168826-7ded1580-c035-11e9-86bf-b5a824ccc23b.png)

**Comment by ghost** [comment]
Im using very cropped PrestaShop as "PrestaCMS" for electronics manufacturer.
We have a lot of products and list of every attachment looks bad.

![list](https://user-images.githubusercontent.com/8243273/64521488-5e759f80-d2f8-11e9-84fd-218712f6aa0f.jpg)


1.6 approach was much better.
With addition of language selection will be perfect.

Keep a good work guys! :)

**Comment by TristanLDD** [comment]
@marionf Yes the dropdown with languages would be the most consistent option :)

**Comment by marionf** [comment]
The problem that I see is if you have the same file for X languages, you will have to add it X times for each

**Comment by TristanLDD** [comment]
The problem could be solved by adding checkboxes in the language dropdown so you can select X languages for one file. This component doesn't exist in the UI kit yet but would be useful. 
I'll work on the mockups along with the other improvements of the product page.

**Comment by marionf** [comment]
I just thought that the language selector on the product page is at the top and applies for the whole page

**Comment by TristanLDD** [comment]
Yes but the problem is that with that selector only it's not possible to know which fields are multi language and which fields are not.

**Comment by Hlavtox** [comment]
Not relevant anymore, old product page is gone.
